controllers/capabilities/tenant_threescale_reconciler.go

```diff
@@ -309,3 +314,36 @@ func (r *TenantThreescaleReconciler) retrieveTenantID() (int64, error) {
 
 	return tenantId, nil
 }
+
+func (r *TenantThreescaleReconciler) SetUpdateTenantInfo(tenant *porta_client_pkg.Tenant) error {
+	// set/update Tenant optional Info parameters
+	params := map[string]string{}
+	if r.tenantR.Spec.FromEmail != nil {
```

Comment:
WDYT if instead of making the update call on every reconcile we would compare the tenantDef against tenant CR and see if the fields are already set, if so, do not make the call?

Reply:
Hey @MStokluska , Thank you for comment. Seems to me - two things. 1) consistency with other 3scale-operator's reconcilers, and I see in developeraccount reconciler, that looks like just update according to CR on each cycle 2) The question is interesting, so I tried a bit to look at Google, for operator sdk questions, and see following `The pattern that the Reconcile() method intentionally forces you into is that you should never care if your CR was created or updated. Instead, on every single reconciliation, your reconcile should compare the state of the cluster and the app with the desired state in the CR spec and then reconcile the cluster and the app to match the CR spec's desired state.`
So  ... maybe we could leave it as is WDYT? But if you think that we need check, maybe you could also suggest how, as seems we can try Rest call, account Get, and take all params from json response. But, again, don't know if need to do it. If you want, we maybe can discuss the Operator reconciliation approches in general in Team. It's interesting for me.  Thank you for the comment.

Reply:
After a discussion with Valery, we decided to leave it as it is for now with future update to porta client to also populate the emails fields and improve this in future.

Reply:
This is what I suggested some time ago here https://github.com/3scale/3scale-operator/pull/906#discussion_r1430655392

Reply:
Thanks, we have just discussed this with the team and decided that actually, bumping porta first and then doing this call when required is better.

Reply:
Porta client updated - PR for review https://github.com/3scale/3scale-porta-go-client/pull/59 
3scale operator last commit is working with updated porta-client and comparing tenant and CR